Client/Application/LeanplumIntegration.swift

```diff
@@ -105,9 +105,9 @@ class LeanplumIntegration {
         setupTemplateDictionary()
 
         var userAttributesDict = [AnyHashable: Any]()
-        userAttributesDict[UserAttributeKeyName.alternateMailClient] = "mailto:"
-        userAttributesDict[UserAttributeKeyName.focusInstalled] = !canInstallFocus()
-        userAttributesDict[UserAttributeKeyName.klarInstalled] = !canInstallKlar()
+        userAttributesDict[UserAttributeKeyName.alternateMailClient.rawValue] = "mailto:"
+        userAttributesDict[UserAttributeKeyName.focusInstalled.rawValue] = !canInstallFocus()
+        userAttributesDict[UserAttributeKeyName.klarInstalled.rawValue] = !canInstallKlar()
```

Comment:
Another option is to make your `enum` a `CustomStringConvertable`.

```
enum Foo: String, CustomStringConvertable {
    case foo = "Foo"
}
var description: String {
    return self.rawValue
}
```

More Swifty.